plugin/adsk/scripts/AEmayaUsdProxyShapeTemplate.mel

```diff
@@ -1,11 +1,204 @@
+        editorTemplate -suppress "initialSampleRate";
+        editorTemplate -suppress "extraSampleRate";
+        editorTemplate -suppress "textureThreshold";
+        editorTemplate -suppress "normalThreshold";
+        editorTemplate -suppress "boundingBoxScale";
+
+        // Render Stats (completely remove):
+        editorTemplate -suppress "castsShadows";
+        editorTemplate -suppress "receiveShadows";
+        editorTemplate -suppress "holdOut";
+        editorTemplate -suppress "motionBlur";
+        editorTemplate -suppress "primaryVisibility";
+        editorTemplate -suppress "smoothShading";
+        editorTemplate -suppress "visibleInReflections";
+        editorTemplate -suppress "visibleInRefractions";
+        editorTemplate -suppress "doubleSided";
+        editorTemplate -suppress "checkDoubleSided";
+        editorTemplate -suppress "opposite";
+        editorTemplate -suppress "geometryAntialiasingOverride";
+        editorTemplate -suppress "checkGeomAntialiasingOverride";
+        editorTemplate -suppress "antialiasingLevel";
+        editorTemplate -suppress "shadingSamplesOverride";
+        editorTemplate -suppress "checkGeomShadingSamplesOverride";
+        editorTemplate -suppress "shadingSamples";
+        editorTemplate -suppress "maxShadingSamples";
+        editorTemplate -suppress "volumeSamplesOverride";
+        editorTemplate -suppress "volumeSamples";
+        editorTemplate -suppress "depthJitter";
+
+        // 3D Motion Blur (completely remove):
+        editorTemplate -suppress "maxVisibilitySamplesOverride";
+        editorTemplate -suppress "maxVisibilitySamples";
+
+        // Ghosting Information (completely remove):
+        editorTemplate -suppress "ghosting";
+        editorTemplate -suppress "ghostingMode";
+        editorTemplate -suppress "ghostingControl";
+        editorTemplate -suppress "ghostCustomSteps";
+        editorTemplate -suppress "ghostsBefore";
+        editorTemplate -suppress "ghostsAfter";
+        editorTemplate -suppress "ghostsStep";
+        editorTemplate -suppress "ghostFrames";
+        editorTemplate -suppress "ghostOpacityRange";
+        editorTemplate -suppress "ghostFarOpacity";
+        editorTemplate -suppress "ghostNearOpacity";
+        editorTemplate -suppress "ghostColorPre";
+        editorTemplate -suppress "ghostColorPreR";
+        editorTemplate -suppress "ghostColorPreG";
+        editorTemplate -suppress "ghostColorPreB";
+        editorTemplate -suppress "ghostColorPreA";
+        editorTemplate -suppress "ghostColorPost";
+        editorTemplate -suppress "ghostColorPostR";
+        editorTemplate -suppress "ghostColorPostG";
+        editorTemplate -suppress "ghostColorPostB";
+        editorTemplate -suppress "ghostColorPostA";
+        editorTemplate -suppress "ghostDriver";
+        editorTemplate -suppress "ghostUseDriver";
+        editorTemplate -suppress "ghostRangeStart";
+        editorTemplate -suppress "ghostRangeEnd";
```

Comment:
Some of these (including these two) are from Maya 2020 (either removed or renamed). If an attribute doesn't exist the suppress/addControl will silently return.